cmd/serv.go

```diff
@@ -143,6 +158,10 @@ func runServ(c *cli.Context) error {
 	username := strings.ToLower(rr[0])
 	reponame := strings.ToLower(strings.TrimSuffix(rr[1], ".git"))
 
+	if setting.EnablePprof {
```

Comment:
I think it should be also allowed to be specified from cli so that it would be possible to profile only single key by editing authorized_keys file

Reply:
I agree, how would that look like then ?

Reply:
Now I have:

```
command="/Users/jerry/src/code.gitea.io/gitea/gitea serv key-1 --config='/Users/jerry/src/code.gitea.io/gitea/custom/conf/app.ini'"
```

Then it would become:

```
command="/Users/jerry/src/code.gitea.io/gitea/gitea serv key-1 --config='/Users/jerry/src/code.gitea.io/gitea/custom/conf/app.ini'",environment="ENABLE_PPROF=true"
```

Which must work out of the box with OpenSSH right?

Reply:
Now I understand, latest commit has it added.